src/java.base/share/classes/java/util/IdentityHashMap.java

```diff
@@ -1382,6 +1382,50 @@ public void replaceAll(BiFunction<? super K, ? super V, ? extends V> function) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * <p>More formally, if this map contains a mapping from a key
+     * {@code k} to a value {@code v} such that {@code key == k}
+     * and {@code value == v}, then this method removes the mapping
+     * for this key and returns {@code true}; otherwise it returns
+     * {@code false}.
```

Comment:
The API documentation of containsKey() and containsValue() should probably be updated to mention reference equality too. This doesn't need to be carried in this PR, but maybe a new issue should be logged to double check the completeness of the IdentityHashMap API documentation and see where adding some similar text makes sense.